DEV-README.md

```diff
@@ -93,4 +177,35 @@ SosParsingFunctions("TimeseriesKVP" = function() { ... })
 
 ### How to add an encoder
 
-...
\ No newline at end of file
+...
+
+
+## Releases
+
+A new release shall be uploaded to CRAN after testing and under the following procedure:
+
+- Run the [tests](#tests)
+- Check the CI status
+  - https://travis-ci.org/52North/sos4R
+  - https://ci.appveyor.com/project/nuest/sos4r
+- Update version and date in `man/sos4R-package.Rd`
+- Update version in `DESCRIPTION`
+- Create a git tag with the version number
```

Comment:
```suggestion
- Create a git tag with the version number in the format `vx.y.z`, e.g `v3.5.56` (following the rules of [semantic versioning](https://semver.org/)
```

Reply:
Good idea to give an example, semantic version is linked to above already, slightly rephrased it.